exclusion.dic

```diff
@@ -0,0 +1,14 @@
+awaiter
+crossgen
+cshtml
+deserializer
+nuget
+nupkg
+nuspec
+sdk
+sdks
```

Comment:
```suggestion
sdks
msbuild
```

Reply:
I'll add it in alphabetical order. One sec.